### DISCUSSION TITLE
Interface connector between Sizif and add on board

### DISCUSSION THREAD

**Discussion by Mr-JimmyDeath** [UzixLS/zx-sizif-512#28] [PCB Build]
In the build list for add on sound card a two row 1.27mm pitch connector is specified. Whilst it is easy to fit the female connector to the Sizif itself I can't fit the male connector to the sound card since the PCB at 1.6mm thick won't fit between the rows unless I bend them which a) looks horrible b) means the plastic part of the connector does not sit completely in the pcb cut out.

Did you use a special connector for this (I can't tell from the photos)? I can make it work nicely if I use single row connectors top and bottom on both the sizif and sound card but then it would not mate properly with a board that previously had a two row connector fitted.

Thanks, James

**Comment by UzixLS** [comment]
I used these ones: https://aliexpress.ru/item/32693979288.html

**Reply by Mr-JimmyDeath** [reply]
Thanks for the quick response

Regards

James
